### PR TITLE
Documented current Private Cloud limitations

### PR DESCRIPTION
**Pull request by zlogic**
To reduce support questions, I've tried to summarize what are the current limitations of Mendix for Private Cloud, and also prerequisites which are confirmed to be supported.

### CODE REVIEW

File: content/developerportal/deploy/private-cloud-supported-environments.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+            ]
+        }
+    ]
+}
+```
+
+## 6 Networking
+
+{{% alert type="info" %}}
+DNS, load balancing and the ingress controller should be configured separately, on a cluster-wide scope.
```
Comment (MarkvanMents) [inline content/developerportal/deploy/private-cloud-supported-environments.md:226]: @zlogic Do you mean that you have to configure them outside the Mendix scope, or do you mean that you need to configure each of the three separately? If the former then perhaps:
```suggestion
DNS, load balancing and the ingress controller should be configured first for the whole Kubernetes cluster.
```

Comment (zlogic) [reply content/developerportal/deploy/private-cloud-supported-environments.md]: @MarkvanMents Your understanding is correct, it's outside of the Mendix scope and is the customer's responsibility.

**Comment by MarkvanMents** [comment]
@zlogic Can this be published now, or does someone else need to approve?

**Comment by zlogic** [comment]
@MarkvanMents Yes, can you please publish this?

**Pull request merged**